zeppelin-web/src/components/login/login.controller.js

```diff
@@ -39,5 +39,15 @@ angular.module('zeppelinWebApp').controller('LoginCtrl',
       });
 
     };
+
+    $rootScope.$on('initLoginValues', function() {
```

Comment:
@r-kamath @corneadoug Thank you for reviewing.
and I totally agree with @corneadoug. I just updated it.